### PR TITLE
Rook toolbox container build and readme instructions

### PR DESCRIPTION
**Pull request by jbw976**
Any feedback is welcome, especially if the fundamental approach could be improved.

Addresses #343

### CODE REVIEW

File: toolbox/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Then launch the toolbox as usual:
+```
+toolbox
+```
+
+### Ceph Tools
+To use the ceph tools from a rook host, launch the toolbox and run the following command:
+```
+toolbox --bind=/var/lib/rook:/var/lib/rook
+ROOK_CONFIG=`find /var/lib/rook -regex '.*mon[0-9]+/.*\.config' | head -1`
```
Comment (jbw976) [inline toolbox/README.md]: an easy to implement but better experience here would be a simple entrypoint script that gets or generates a rook/ceph cluster config file into the default spot that `ceph` and `rados` commands look for (i believe `/etc/ceph`) in the container, so the user doesn't have to deal with anything beyond running the container and then running `ceph` or `rados` with no additional args

Comment (travisn) [inline toolbox/README.md]: what if we change our default cluster name to be the ceph default "ceph"? Using the defaults just makes using all the tools easier.

Comment (jbw976) [inline toolbox/README.md]: i'm not sure there's a super compelling reason for us to have to change our config file name.  With the latest commit the toolbox does everything for a user of it, so they don't even have to think about the config file: https://github.com/rook/rook/pull/381/commits/fb70cb9f335268d0cbb7ae331c954b9fa0b8baa3

**Comment by jbw976** [comment]
i like the idea of having a k8s podspec so that the toolbox container can be run in a pod.  I've captured that in #387 since I have some ramp up to do to get running/functional in the newer k8s environments.

**Pull request merged**